CONTRIBUTING.rst

```diff
@@ -7,6 +7,9 @@ Contributing
 #. Make sure that your commit messages clearly describe the changes.
 #. Send a pull request. (Please Read: `Faster Pull Request Reviews`_)
 
+**Important** All of the client libraries are automatically generated using **gapic-generator-python**.
+Open source contributions for autogenerated code should be made in https://github.com/googleapis/gapic-generator-python.
+
```

Comment:
Thanks for the contribution @antvega98! I've proposed a slight edit to the wording. I also updated the links to follow the same format that we have in for links in https://raw.githubusercontent.com/googleapis/google-cloud-python/main/README.rst

```suggestion
**Note:** All client libraries are automatically generated using `gapic-generator-python`_. Open source contributions for autogenerated code should be made in `gapic-generator-python`_.

.. _gapic-generator-python: https://github.com/googleapis/gapic-generator-python
```